website/docs/cloud-docs/agents/agents.mdx

```diff
@@ -95,6 +95,16 @@ You can configure the agent to accept forwarded requests from HCP Terraform. Req
 
 Agents handle forwarded requests separately from other workloads and may process requests in parallel to plans, applies, policy checks, etc. You can modify this behavior by enabling or disabling certain workload types via the `-accept` parameter, and selectively setting the `-request-forwarding` flag on certain agent(s) only. For example, you may have a pool of 4 agents, where two are configured to handle only plans and applies, and the other two are configured to handle only request forwarding.
 
+
+| Job type name | Description                                                                                                     | Terraform CLI equivalent |
```

Comment:
So, the page does describe the [allowed job types below](https://developer.hashicorp.com/terraform/cloud-docs/agents/agents#accept), but I agree a table could be a better way to display this! This doesn't feel like quite the place for this information though, since it doesn't really relate to request forwarding.

What do you think about placing it on the [Manage agent pool](https://developer.hashicorp.com/terraform/cloud-docs/agents/agent-pools#view-agent-statuses) page? If you like that, could you write write descriptive sentences for each `Description`, instead of using links? In docs we try to have all the information on one page if we can to avoid folks having to open many tabs to get the context they need.